megamek/src/megamek/common/equipment/ArmorType.java

```diff
@@ -251,6 +255,29 @@ public int getBAR() {
         return bar;
     }
 
+    private static ArmorType createUnknownArmor() {
```

Comment:
We used to have an "Unknown" armor type, which is still used in some tests / files.  This should cover that option.